src/pool/cli.c

```diff
@@ -1456,14 +1475,18 @@ pool_query_cb(tse_task_t *task, void *data)
 {
 	struct pool_query_arg	       *arg = (struct pool_query_arg *)data;
 	struct pool_buf		       *map_buf = arg->dqa_map_buf;
-	struct pool_query_v4_in	       *in = crt_req_get(arg->rpc);
-	struct pool_query_v4_out       *out = crt_reply_get(arg->rpc);
+	struct pool_query_v5_in	       *in = crt_req_get(arg->rpc);
```

Comment:
Does this work with old server?

Reply:
yes, v4 and v5 share the same query req(query_in) format, only different on query out.